crates/transaction-pool/src/traits.rs

```diff
@@ -44,10 +44,7 @@ pub type PeerId = alloy_primitives::B512;
 #[auto_impl::auto_impl(&, Arc)]
 pub trait TransactionPool: Send + Sync + Clone {
     /// The transaction type of the pool
-    type Transaction: PoolTransaction<
-        Pooled = PooledTransactionsElementEcRecovered,
-        Consensus = TransactionSignedEcRecovered,
-    >;
+    type Transaction: EthPoolTransaction;
```

Comment:
@emhane I've named the PR because this is specifically the trait bound I am attempting to relax

Reply:
sure, that's not incorrect, though more helpful would be clarifying it's the trait bound on the associated type `TransactionPool::Transaction` in the title